src/main/drivers/dshot_command.c

```diff
@@ -151,18 +151,23 @@ static bool allMotorsAreIdle(void)
     return true;
 }
 
-bool dshotCommandsAreEnabled(void)
+bool dshotCommandsAreEnabled(dshotCommandType_e commandType)
```

Comment:
Should we change this to `dshotInlineCommadsAreEnabled()` so that we can avoid the parameter? I don't think there will ever be need to check this condition for blocking commands.

Reply:
Based on your suggestion this now also checks that motors are disabled before a blocking command is allowed. So it's name is still appropriate as it works for both command types.

Reply:
I meant to suggest to check this inside `dshotCommandWrite()` - apologies for not making myself clear - do you think `dshotCommandsAreEnabled()` will be needed in conjunction with blocking Dshot commands?

Reply:
I think its pretty clean as is. To me it makes sense that `dshotCommandsAreEnabled()` should return the result based on the differing conditions for the different command types. So this locks out "blocking" types if the motors are enabled, and only allows inline types if the motors are enabled (and other guard time conditions). So this would also prevent any unintended use of blocking type of commands in a setting where the motors are running.

Reply:
And remember, `dshotCommandWrite()` calls `dshotCommandsAreEnabled()` and performs an early return if not true.